client/src/components/questionsAnswers/A-QnaWidget.jsx

```diff
@@ -72,14 +72,13 @@ function QnaWidget(props) {
   }
 
   return (
-    <div>
+    <div className="qnaApp">
```

Comment:
Class names should be kebab-case. This won't affect us, however it is the standard to use kebab-case: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/